packages/webapp/public/locales/es/translation.json

```diff
@@ -1711,102 +1711,99 @@
+    "MODEL": "Modelo",
+    "MODEL_HELPTEXT": "Este id se utiliza para identificar de forma única este sensor en otros sistemas integrados y no se puede cambiar. Si ya no se usa, intente retirar este sensor y agregar uno nuevo.",
+    "EXTERNAL_IDENTIFIER": "Identificador externo",
+    "PART_NUMBER": "Número de parte",
+    "HARDWARE_VERSION": "Versión de hardware",
+    "NO_DATA_FOUND": "No se encontraron lecturas del sensor",
+    "NO_DATA": "(sin datos)",
     "DETAIL": {
-      "RETIRE": "MISSING",
-      "EDIT": "MISSING",
-      "NAME": "MISSING",
-      "LATITUDE": "MISSING",
-      "LONGITUDE": "MISSING",
-      "MODEL": "MISSING",
-      "PART_NUMBER": "MISSING",
-      "HARDWARE_VERSION": "MISSING",
-      "EXTERNAL_ID_TOOLTIP": "MISSING",
-      "EXTERNAL_IDENTIFIER": "MISSING",
-      "DEPTH": "MISSING",
-      "BRAND": "MISSING",
-      "BRAND_TOOLTIP": "MISSING",
-      "READING_TYPES": "MISSING"
+      "RETIRE": "Retirar",
+      "EDIT": "Editar",
+      "NAME": "Nombre del sensor",
+      "LATITUDE": "Latitud",
+      "LONGITUDE": "Longitud",
+      "MODEL": "Modelo",
+      "PART_NUMBER": "Número de pieza",
+      "HARDWARE_VERSION": "Versión de hardware",
+      "EXTERNAL_ID_TOOLTIP": "Esta identificación se utiliza para identificar de forma única este sensor en otros sistemas integrados y no se puede cambiar. Si ya no se usa, retire este sensor y agregar uno nuevo.",
+      "EXTERNAL_IDENTIFIER": "Identificador externo      ",
+      "DEPTH": "Produndidad",
+      "BRAND": "Marca",
+      "BRAND_TOOLTIP": "A continuación se muestran las marcas con las que LiteFarm puede integrarse. Si ya no desea utilizar esta marca de sensores, retire este sensor.",
+      "READING_TYPES": "Tipos de lectura"
     },
     "VIEW_HEADER": {
-      "READINGS": "MISSING",
-      "TASKS": "MISSING",
-      "DETAILS": "MISSING"
+      "READINGS": "Lecturas",
+      "TASKS": "Tareas",
+      "DETAILS": "Detalles"
     },
     "RETIRE": {
-      "TITLE": "MISSING",
-      "RETIRE": "MISSING",
-      "CANCEL": "MISSING",
-      "BODY": "MISSING",
-      "RETIRE_SUCCESS": "MISSING",
-      "RETIRE_FAILURE": "MISSING"
+      "TITLE": "¿Retirar sensor?",
+      "RETIRE": "Retirar",
+      "CANCEL": "Cancelar",
+      "BODY": "Al retirar este sensor, se eliminará de su granja y se cerrarán las conexiones con el proveedor del sensor. ¿Quieres seguir adelante?",
+      "RETIRE_SUCCESS": "Sensor retirado con éxito",
+      "RETIRE_FAILURE": "Hubo un error al retirar este sensor"
     },
     "TEMPERATURE_READINGS_OF_SENSOR": {
-      "TITLE": "MISSING",
-      "SUBTITLE": "MISSING",
-      "Y_AXIS_LABEL": "MISSING",
-      "WEATHER_STATION": "MISSING",
-      "NO_DATA": "MISSING",
-      "AMBIENT_TEMPERATURE_FOR": "MISSING",
-      "LAST_UPDATED_TEMPERATURE_READINGS": "MISSING"
+      "TITLE": "Temperatura del suelo",
+      "SUBTITLE": "Temperatura ambiente alta y baja de hoy: {{alta}}{{tempUnit}} / {{baja}}{{tempUnit}}",
```

Comment:
Please keep the texts in `{{ }}` (high and low) :)
English: `"Today’s ambient high and low temperature: {{high}}{{tempUnit}} / {{low}}{{tempUnit}}"`

Reply:
This is translated. High is "alta" and Low is "baja". Do I also need to translate tempUnit?